src/screens/Main/Profile/profile.screen.tsx

```diff
@@ -1,36 +1,21 @@
 import { FlatList, Image, TouchableOpacity, View } from "react-native";
-import { FC, useState } from "react";
+import { FC } from "react";
 import { NativeStackScreenProps } from "@react-navigation/native-stack";
 import { NavigatorParamList } from "navigators";
 import { colors, typography } from "theme";
-import { StatusModal, Text } from "components";
+import { Text } from "components";
 import { hp, wp } from "utils/responsive";
 import { MY_PROFILE_DATA } from "constant";
-import { UserStatusI } from "../Home/home.screen";
+import { RootState, useAppSelector } from "../../../store/store";
+import noImage from "../../../assets/images/noImage.jpeg";
```

Comment:
use absolute path. Make sure to use absolute path in all files. Check every file now before moving to another comment!